.changeset/sweet-masks-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': minor
```

Comment:
I think this needs to be a patch so that it continues to be `next`-releases of 1.0.0

Reply:
Ah yes i see, can i just edit the generated md file or delete and create a new changeset